.github/workflows/cancel-old-reviews.yaml

```diff
@@ -0,0 +1,17 @@
+on:
+    schedule:
+        -   cron: '0 2 * * *'
+name: Cancel reviews older than seven days
+jobs:
+    rebuild-master-images:
+        name: Cancel reviews older than seven days
+        runs-on: [ self-hosted, linux, review-stage ]
+        steps:
+            -   name: GIT checkout branch - master
+                uses: actions/checkout@v3
+                with:
+                    ref: 'refs/heads/master'
```

Comment:
What about just curl the required file instead of the full checkout of the repository?

Reply:
🤔  the `manually_cancel_review` workflow is written without checkout or curl... is it necessary here at all?

Reply:
Well, I need that file :D do you have a better idea?